### ISSUE TITLE
EBU STL: "{\an8}" in front of every title if VP=2

### ISSUE THREAD

**Issue by mivk** [SubtitleEdit/subtitleedit#325]
When importing some EBU STL subtitles, all subtitles start with `{\an8}`.
I understand this is a positioning code (from the ASS standard?), and I think it is supposed to mean "top center".

The EBU STL files where I have seen this have the VP ("Vertical Position") byte set to 2 on each title, and the DSC (Display Standard Code) byte in the header set to 0 ("Open subtitling").

The problem is that this positioning code becomes part of the subtitle text, and is exported as such in all other formats.

For example, exporting to a plain text format like Avid Captions gives:

```
@ This file written with the Avid Caption plugin, version 1

<begin subtitles>
00:59:59:00 00:59:59:01
{\an8}Beep

01:00:30:17 01:00:34:22
{\an8}Neuf ans après ...
```

The positioning information should not be displayed as part of the text, or at least it should be stripped when saving to formats which don't support it.

**Comment by boriseetgerink** [comment]
This looks like it's still an issue in https://github.com/SubtitleEdit/subtitleedit/blob/9b81bc046be7333f31deffd7e90a3f1e8b5851cd/libse/SubtitleFormats/Ebu.cs

I'll compare the implementation to the specification (https://tech.ebu.ch/docs/tech/tech3264.pdf) when I have the time. Hopefully sometime next week.

**Comment by boriseetgerink** [comment]
For me this occurred in version 3.4.10 when converting STL to 890. The STL had a vertical position 1 and justification code 2, which means display top center. This explains why `{\an8}` was inserted.

Reconverting the STL to 890 using version 3.4.13 fixed this issue - the tag isn't inserted in the 890 version. I'll have a look why this is fixed.

**Comment by boriseetgerink** [comment]
This looks like its fixed in 2731b43a and 10179c2d, which are included in version 3.4.13
Looks like this issue can be closed.

**Comment by niksedk** [comment]
thx for the info :)
